.github/workflows/build_and_push_docker_img.yml

```diff
@@ -1,9 +1,10 @@
-name: Build and push the docker image to DockerHub
+name: Pull API code, Build and push the docker image to DockerHub
 
 on:
   push:
     branches:
       - main
+      - bita
```

Comment:
Let's only push the image from main branch so that the experimental new features are not added to docker image yet.

Reply:
I pushed my changes Arash, and I tested them. For the demo, please run the GitHub workflow to update the docker image from the main branch. I can only rerun jobs on my branch. Please review and merge next week.